tee-worker/litentry/core/data-providers/src/graphql.rs

```diff
@@ -264,6 +265,224 @@ pub struct TotalTxsStruct {
 	pub total_transactions: u64,
 }
 
+/// AchainableTag: Account
+/// base_url: https://label-production.graph.tdf-labs.io
+/// test key: 26353d4c-b01c-4466-98a5-80d3fc53a9d8
+pub trait AchainableTagAccount {
+	fn fresh_account(&mut self, address: &str) -> Result<bool, Error>;
+	fn og_account(&mut self, address: &str) -> Result<bool, Error>;
+	fn class_of_2020(&mut self, address: &str) -> Result<bool, Error>;
+	fn class_of_2021(&mut self, address: &str) -> Result<bool, Error>;
+	fn class_of_2022(&mut self, address: &str) -> Result<bool, Error>;
+	fn found_on_bsc(&mut self, address: &str) -> Result<bool, Error>;
+	fn is_polkadot_alidator(&mut self, address: &str) -> Result<bool, Error>;
+	fn is_kusama_validator(&mut self, address: &str) -> Result<bool, Error>;
+}
+
+impl AchainableTagAccount for GraphQLClient {
+	fn fresh_account(&mut self, address: &str) -> Result<bool, Error> {
+		let params = ReqParams::new("/v1/run/label/1de85e1d215868788dfc91a9f04d7afd");
+		let body = ReqBody { params: ParamsAccount { address: address.to_string() } };
+		let response =
+			self.client.post_capture::<ReqParams, ReqBody, serde_json::Value>(params, &body);
+		debug!("ReqBody response: {:?}", response);
```

Comment:
These functions differs only in request's params and body, have you considered extracting the common part ?

Reply:
Yes, there are only slight differences between them. Subsequent pr will include all Balance-related interfaces, and the corresponding refactoring is planned on that pr.